runtime/pendulum/Cargo.toml

```diff
@@ -194,34 +211,67 @@ std = [
   "sp-std/std",
   "sp-transaction-pool/std",
   "sp-version/std",
-  "vesting-manager/std",
   "xcm-builder/std",
   "xcm-executor/std",
   "xcm/std",
   "zenlink-protocol/std",
   "zenlink-protocol-runtime-api/std",
   "bifrost-farming/std",
   "bifrost-farming-rpc-runtime-api/std",
-  "orml-currencies-allowance-extension/std"
+
+  #custom libraries from spacewalk
+    "security/std",
+    "staking/std",
+    "oracle/std",
+    "stellar-relay/std",
+    "fee/std",
+    "vault-registry/std",
+    "redeem/std",
+    "issue/std",
+    "currency/std",
+    "nomination/std",
+    "replace/std",
+    "module-issue-rpc-runtime-api/std",
+    "module-oracle-rpc-runtime-api/std",
+    "module-redeem-rpc-runtime-api/std",
+    "module-replace-rpc-runtime-api/std",
+    "module-pallet-staking-rpc-runtime-api/std",
+    "module-vault-registry-rpc-runtime-api/std",
+    "spacewalk-primitives/std",
+
+    # custom libraries from pendulum
+	  "orml-currencies-allowance-extension/std",
+	  "parachain-staking/std",
+	  "vesting-manager/std",
```

Comment:
Now we have three different levels of spacing here.

Reply:
It is so strange because in my editor I don't see the different indentation! Last time it happened the same, now I can see what you mean.